<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ajopoyta</class>
 <widget class="QWidget" name="Ajopoyta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="nopeusSaadin">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>21</width>
     <height>361</height>
    </rect>
   </property>
   <property name="maximum">
    <number>220</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>50</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
   <property name="tickInterval">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="nopeusLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>000 </string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="stopNappi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ratapiha-kauko.qrc">
     <normaloff>:/r/pic/stop.png</normaloff>:/r/pic/stop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ajopoyta2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ajopoyta1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="etsi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="ratapiha-kauko.qrc">
     <normaloff>:/r/pic/fit-best.png</normaloff>:/r/pic/fit-best.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="jkvTieto">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>150</width>
     <height>300</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(0, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="automaatioNappi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tavoiteNopeusLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>29</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>000</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>km/h</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="veturiValinta">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="veturinTyyppi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>jkvTieto</zorder>
  <zorder>nopeusSaadin</zorder>
  <zorder>nopeusLabel</zorder>
  <zorder>stopNappi</zorder>
  <zorder>ajopoyta2</zorder>
  <zorder>ajopoyta1</zorder>
  <zorder>etsi</zorder>
  <zorder>automaatioNappi</zorder>
  <zorder>tavoiteNopeusLabel</zorder>
  <zorder>label</zorder>
  <zorder>veturiValinta</zorder>
  <zorder>veturinTyyppi</zorder>
 </widget>
 <resources>
  <include location="ratapiha-kauko.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>nopeusSaadin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>tavoiteNopeusLabel</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
